packages/components/src/dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,85 @@
+import { useState, ReactNode } from 'react';
+import React from 'react';
+import { type DropdownProps } from "@/dropdown/Dropdown.types";
+import { globalStyles } from "@/index.styled";
+import { DropdownContainer, DropdownLabel, DropdownChildren, DropdownItem, darkTheme } from "@/dropdown/Dropdown.styled"; 
+import chevronDown from "@/assets/chevron-down.svg"
+
+export const Dropdown = ({
+        label = "Select...", 
+        theme, 
+        showingOptions: initialShowingOptions, 
+        disabled, 
+        error, 
+        children}: DropdownProps) => {
+
+    globalStyles();
+
+    const [showingOptions, setShowingOptions] = useState(initialShowingOptions);
+    const [selectedOption, setSelectedOption] = useState('');
```

Comment:
The 'selectedOption' prop should be a prop and make it an optional prop. Another possibility is to change the name to something simpler, such as 'selected' or 'initialValue'. At the same time, you probably want to add a new prop i.e 'setValue'/'onChange' to allow others to access the dropdown's value